browser/resources/settings/shortcuts_page/components/ConfigureShortcut.tsx

```diff
@@ -66,7 +37,8 @@ const HintText = styled.div`
 `
 
 const InUseAlert = styled(Alert)`
-  margin: ${spacing[24]};
+  margin-top: ${spacing[24]};
+  display: block;
```

Comment:
Note: The <Alert> is currently display: inline.

Reply:
> @simonhong I just fixed a margin on the `<InUseAlert>`. Mind taking another glance?

Still good for me! It would great to update above screenshot :)